pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -112,7 +114,13 @@ stdenv.mkDerivation (finalAttrs: {
     "shadowstack"
   ];
 
-  nativeBuildInputs = [ cmake ]
+  nativeBuildInputs = [
+    cmake
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [
```

Comment:
Do you want this to be conditional like this?

Reply:
ah, whoops, that was a mass-rebuild mitigation. One sec